low-level/framework/messaging/ingress_processor.py

```diff
@@ -185,31 +185,44 @@ def _process_msg(self, body):
                     "IngressProcessor, Authentication failed on message: %s" % ingressMsg)
                 return
 
+            # Check for debugging being activated in the message header
+            self._check_debug(message)
```

Comment:
use logging function like everywhere else, this was legacy way
logger.debug doesnt suffice ?

Reply:
Changed to `logger.debug()`
We have an attribute `debug_enabled` in actuator request to indicate whether to enable debug logging or not.
`_check_debug()` function checks and enable that attribute.
Sample actuator request:
```
{
  "title": "SSPL-LL Actuator Request",
  "description": "Seagate Storage Platform Library - Low Level - Actuator Request",
  "username": "sspl-ll",
  "expires": 3600,
  "signature": "None",
  "time": "2018-07-31 04:08:04.071170",
  "message": {
    "sspl_ll_debug": {
      "debug_component": "sensor",
      "debug_enabled": true
    },
    "sspl_ll_msg_header": {
      "msg_version": "1.0.0",
      "uuid": "16476007-a739-4785-b5c7-f3de189cdf9d",
      "schema_version": "1.0.0",
      "sspl_version": "1.0.0"
    },
"request_path": {
      "site_id": "1",
      "rack_id": "1",
      "node_id": "1"
    },
    "response_dest": {},
    "target_node_id":"1",
    "actuator_request_type": {
      "service_controller": {
      "service_request": "status",
      "service_name": "kafka.service"
       }
    }
  }
}
```

Reply:
> sample request looks fine, except below part. target_node_id is the only new field, mandatory one. rest remains same i believe

Yes, only `target_node_id` is added, rest is same.
>basically no Low level reference should be there in any code atleast

`low level` is not present in code, removed from the comment here as well.

Reply:
>Yeah had seen that, we dont use that any more in new requests atleast, all log levels managed by set log level config & not through request

Should I remove `_check_debug(message)` function?

Reply:
remove its usage from here, thats enough. rest of the cleanup we will take up separately

Reply:
Removed `_check_debug(message)`